### ISSUE TITLE
Add RadiusNeighborsClassifier

### ISSUE THREAD

**Issue by sanketsarang** [blobcity/autoai#48]
Add RadiusNeighborsClassifier model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/classifier_config.py

Reference RadiusNeighborsClassifier Implementation:
https://github.com/blobcity/ai-seed/blob/main/Classification/Radius%20Neighbors/RadiusNeighborsClassifier.ipynb

Dependencies if any, must be appropriately added. Test run of the train function on a classification problem must pass, and the function must attempt to train a RadiusNeighborsClassifier as a potential best fit model.

**Comment by aadityasinha-dotcom** [comment]
Can I work on this issue?

**Comment by sanketsarang** [comment]
Done, it's assigned to you @aadityasinha-dotcom

**Comment by aadityasinha-dotcom** [comment]
We have to directly add the model and its Tuning parameters just like KNeighborsClassifiers right?

**Comment by sanketsarang** [comment]
Yes, that is correct. Just make sure you do a test run using a Classification dataset before the PR.
